### DISCUSSION TITLE
Comments on website

### DISCUSSION THREAD

**Discussion by hpvd** [terminusdb/terminusdb#1874] [Ideas]
Just looked at your website "as a stranger"
it looks really great and professional!

1) There are some tiny ideas to optimize the first impression (and make people spend more time to understand what you have)
- "`Github`" is somehow the synonym for openness and opensource, so maybe you would like to add its symbol and name to the very first row (e.g. directly into the head, just compare to other  projects), otherwise you only see pricing  and it looks "only" commercial
- add details to Community menu flyout to make it even more welcoming (github, discord... like in TerminusCMS-Menu)
- on community site: Add Github icon be more easy visible above the fold (look at website on a small screen or on your smartphone - without scrolling), e.g. within "Github Repo" Button
- add Blog categories to Blog menu flyout (you have really great ones: show it!)

2) a bigger one at the end:

I totally agree that your current customers may use your DB as (advanced) CMS and that's why you have extended it to a new product `TerminusCMS`.

The other site:
you (your website) do not look as a highly specialized serious DB Provider anymore
the issue: You read "CMS" and think of `Wordpress` or what ever (and can't get it out of the head as starting point)
(maybe this is only my personal impression, maybe the majority of people seeing it this way)

Of course, there are descriptions and examples that it is a somehow different/advanced CMS.
But the thing first seen is the name is `TerminusCMS`.
Maybe it's only the name that hides so much... e.g. something like "Terminus the Graph CMS"or"...Data-CMS" would give a totally different expectation what it is and what it can do.
Why can this be important? imho developers and architects are the ones who scale the usage of a specialized product, and not the web agencies who simply customize it for their customers (typical people using CMS)
The essence:
**The word CMS sells it very much below value.**

edit: **this is the total opposite to `Graph-Database with git functionality for Datamesh`** (which I understand as the long term vision)

(of course everything said above is only imho, I apologize if it sounds like "I know it all", it shouldn't. It should only give another pov.)

**Comment by hpvd** [comment]
edited post above.

**Reply by OJ423** [reply]
Hi hpvd, thanks for your feedback, it is greatly appreciated. You make some excellent points. The CMS/DB one is something we grappled with for a while. I completely agree with you regards to the initial thought being `WordPress` and it's something we've tried to negate, but requires too many words to explain (TL;DR!). You've given us some food for thought, so watch this space. Thanks again for taking the time to suggest some improvements.

**Reply by hpvd** [reply]
great to see you seem to like some of the ideas and have already implemented them! Wow what a speed...
